README.md

```diff
@@ -27,4 +27,76 @@ module.exports = {
 };
 ```
 
-For more, see [the Prettier documentation](https://prettier.io/docs/en/configuration.html#sharing-configurations).
+For more information, see [the Prettier documentation](https://prettier.io/docs/en/configuration.html#sharing-configurations).
+
+### With ESLint
+
+`npm install --save-dev eslint-config-prettier`
+
+In your ESLint configuration, add `"prettier"` to `extends`. For example, with `.eslintrc.json` and [the 6 River ESLint configuration](https://github.com/6RiverSystems/eslint-config-6river):
+
+```json
+{
+	"extends": ["6river", "prettier"]
+}
+```
+
+If you are using Typescript and the ESLint plugin `@typescript-eslint/eslint-plugin`, add `"prettier/@typescript-eslint"`
+to `extends` as well. For example, with `.eslintrc.json` and [the 6 River ESLint configuration](https://github.com/6RiverSystems/eslint-config-6river):
+
+```json
+{
+	"extends": ["6river/typescript", "prettier", "prettier/@typescript-eslint"]
+}
+```
+
+For more information, see [the `eslint-config-prettier` documentation](https://github.com/prettier/eslint-config-prettier#installation).
+
+## Usage
+
+### Editor Plugins
+
+The most common way to use Prettier is through an editor plugin that automatically formats files on every save.
+
+There are Prettier plugins for most editors. See the full list on [prettier.io](https://prettier.io/).
```

Comment:
90%+ of the folks here I think use VSCode.  Provide a link to the extension for that.